utils.js

```diff
@@ -26,11 +28,13 @@ var toAbsolute = module.exports.toAbsolute = function(fileOrDirectory) {
     return fileOrDirectory;
   }
 
+  var absFileOrDirectory = fileOrDirectory;
+
   if (!path.isAbsolute(fileOrDirectory)) {
-    fileOrDirectory = path.join(process.cwd(), fileOrDirectory);
+    absFileOrDirectory = path.join(process.cwd(), absFileOrDirectory);
```

Comment:
I guess this could also be better as a ternary:

```js
return path.isAbsolute(fileOrDirectory) ?
  fileOrDirectory :
  path.join(process.cwd(), fileOrDirectory);
```